src/client/components/intro/Intro.js

```diff
@@ -3,6 +3,21 @@ import { SocketContext } from '../../App';
 import Room from '../room/Room';
 import './Intro.css';
 
+const encouragements = [
```

Comment:
why r some of them single quotes while others are double quotes this is deeply disturbing

Reply:
[https://prettier.io/](url)

Reply:
ya link dont work